rpw_impl/ObstacleData[] obstacles